parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -160,6 +160,8 @@ if [ "$IS_IPV6_DUALSTACK_FEATURE_ENABLED" = "true" ]; then
     wait_for_file 3600 1 $dhcpv6_systemd_service || exit $ERR_FILE_WATCH_TIMEOUT
     wait_for_file 3600 1 $dhcpv6_configuration_script || exit $ERR_FILE_WATCH_TIMEOUT
     ensureDHCPv6
+
+    retrycmd_if_failure 120 5 25 modprobe ip6_tables || exit $ERR_MODPROBE_FAIL
```

Comment:
not a blocker for this PR but as a follow up could you refactor the ipv6 cse setup into a function in cse_config.sh ?